backend/user/models.py

```diff
@@ -11,46 +11,52 @@
 
 
 class NotificationToken(models.Model):
-    KIND_IOS = "IOS"
-    KIND_ANDROID = "ANDROID"
-    KIND_OPTIONS = ((KIND_IOS, "iOS"), (KIND_ANDROID, "Android"))
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    token = models.CharField(max_length=255, primary_key=True)
+    class Meta:
+        abstract = True
 
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    kind = models.CharField(max_length=7, choices=KIND_OPTIONS, default=KIND_IOS)
-    token = models.CharField(max_length=255)
+class IOSNotificationToken(NotificationToken):
+    is_dev = models.BooleanField(default=False)
+
+class AndroidNotificationToken(NotificationToken):
+    pass
```

Comment:
would an `is_dev` field not make sense here too? I don't know the Android notif system but would be surprised if they didn't have similar optionality to iOS.

Reply:
after talking to android didn't seem like they had